src/views/tutorial-library/index.tsx

```diff
@@ -65,7 +65,7 @@ export default function TutorialLibraryView() {
 				</Dialog>
 			</div>
 			<CurrentFilters />
-			<TutorialLibraryResults />
+			<TutorialLibraryResults defaultTutorials={defaultTutorials} />
```

Comment:
A little prop drilling action here, but it feels okay as we are able to fetch the data in `getStaticProps`.